extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -269,12 +345,19 @@ export class ConsentUI {
     }
 
     const data = getData(event);
-    if (!data) {
+    if (!data || !data['type']) {
       return;
     }
 
     if (data['type'] == 'consent-ui-ready') {
       this.iframeReady_.resolve();
     }
+
+    if (data['type'] === 'consent-ui-enter-fullscreen') {
```

Comment:
what happens if `consent-ui-enter-fullscreen` is received before `consent-ui-ready`?